src/main/java/com/emc/object/ObjectConfig.java

```diff
@@ -437,6 +439,13 @@ public int getReadTimeout() {
 
     public void setReadTimeout(int readTimeout) {this.readTimeout = readTimeout; }
 
+    @ConfigUriProperty
+    public String getSessionToken() {
+        return sessionToken;
+    }
+
+    public void setSessionToken(String sessionToken) {this.sessionToken = sessionToken; }
```

Comment:
add a javadoc comment here describing what the session token is - something like the following:
> When using temporary S3 credentials, this is the secure session token returned by the STS service